product/validators/accept_purchase_request_validator.py

```
@@ -0,0 +1,21 @@
+from product.services.banned_item_checker import check_item_banned
+
+
+def validate_accept_purchase_request(user, purchase_request_id):
+    """
+    Validates the accept purchase request API requirements.
+    """
+
+    try:
+        purchase_request = PurchaseRequest.objects.get(id=purchase_request_id)
+    except PurchaseRequest.DoesNotExist:
+        raise PurchaseRequestNotFoundException()
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)